data/migrations/20191016080237_user_profiles.js

```diff
@@ -1,7 +1,7 @@
 exports.up = function(knex) {
   return knex.schema
     .createTable("profiles", profiles => {
-      profiles.increments("profile_id").unique();
```

Comment:
were you running into some issues with this being unique? i had a weird bug when i was testing the registration where it was trying to create users with id's that already existed. Either way, we need the id's to be unique so we'll have to figure out how to make it work.